templates/flavors/base/cluster-template.yaml

```diff
@@ -98,6 +98,26 @@ spec:
             "useManagedIdentityExtension": false,
             "useInstanceMetadata": true
           }
+    diskSetup:
+      partitions:
+        - device: /dev/disk/azure/scsi1/lun0
+          tableType: gpt
+          layout: true
+          overwrite: false
+      filesystems:
+        - label: etcd_disk
+          filesystem: ext4
+          device: /dev/disk/azure/scsi1/lun0
+          extraOpts:
+            - "-E"
+            - "lazy_itable_init=1,lazy_journal_init=1"
+        - label: ephemeral0
```

Comment:
ephemeral (/mnt) filesystem is explicitly configured, see: https://bugs.launchpad.net/cloud-init/+bug/1879552